testing/kuttl/e2e/pgbackrest-init/01-pgbackrest-connect.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+# When the cluster comes up, only the repo in the 0th position has activated with a backup,
+# so the pgbackrest status should be "mixed"
+- script: CLUSTER=init-pgbackrest ../../scripts/pgbackrest-initialization.sh "mixed"
+  background: false
```

Comment:
⛏️ The docs say `background` is only meaningful in `TestSuite`. Did you find otherwise?

Reply:
Ah, this was inherited, let me experiment.

Experimenting done! Removed!